src/components/MyHousesList/MyHousesList.module.css

```diff
@@ -0,0 +1,24 @@
+.listContainer {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.listButton {
+  width: 662px;
+}
+
+.listCollapse {
+  width: 662px;
+  color: #535c68;
+}
+
+@media screen and (max-width: 900px) {
```

Comment:
Please consider rewriting this with the mobile first approach so that `min-width` is used instead here.